private/tools/java/com/github/bazelbuild/rules_jvm_external/maven/MavenPublisher.java

```diff
@@ -78,6 +103,10 @@ public static void main(String[] args)
     boolean gpgSign = Boolean.parseBoolean(System.getenv("GPG_SIGN"));
     Credentials credentials =
         new BasicAuthCredentials(System.getenv("MAVEN_USER"), System.getenv("MAVEN_PASSWORD"));
+    boolean useInMemoryPgpKeys = Boolean.parseBoolean(System.getenv("USE_IN_MEMORY_PGP_KEYS"));
```

Comment:
You can use `Boolean.getBoolean`

Reply:
Seems like the existing code are all using `Boolean.parseBoolean`. I prefer to keep using the same api and tackle it as a follow up in case there's any edge case.